tools/InterfaceGenerator/generator/parsers/RPCBase.py

```diff
@@ -555,6 +559,11 @@ def _parse_param_base_item(self, element, prefix):
         
         params["is_mandatory"] = self._get_bool_from_string(is_mandatory)
 
+        deprecated = None
+        deprecated = self._extract_optional_bool_attrib(attrib, "deprecated", False)
+        if deprecated is not None:
```

Comment:
This is guaranteed not to be `None`, since you used a default value of `False` in the line above